http/timer_queue.hpp

```diff
@@ -43,7 +43,6 @@ class TimerQueue
     {
         if (dq.size() == maxSize)
         {
-            BMCWEB_LOG_ERROR << "timer add: dq.size has hit maxSize";
```

Comment:
Still want this message?

Reply:
if (connectionCount >= 100)
        {
            BMCWEB_LOG_CRITICAL << this << "Max connection count exceeded.";
            
            
should cover us. 
This use case isn't really valid anymore.